plaidml/op/op.h

```diff
@@ -5,8 +5,11 @@
 #include <string>
 #include <vector>
 
+#include "llvm/Support/FormatVariadic.h"
```

Comment:
Unfortunately I think this is not a good dependency to introduce to the EDSL public API. This means that all users of the EDSL would have to pull in LLVM.

Reply:
Ok, I'll back it out